stage1.py

```
@@ -1058,24 +916,24 @@ def create_tcp_socket(sc):
             % (listen, sc.errno, sc.get_error_string())
         )
-    debugprint("[*] Listening on socket: %d" % s)
+    print("[*] Listening on socket: %d" % s)
 
     return s
 
 
 def poc():
-    debugprint(
+    print(
         "[*] Detected console kind: %s, game name: %s" % (CONSOLE_KIND, config.name)
```

Ruff (0.14.3)

Undefined name `config`

(F821)